### PR TITLE
Fix: Incorrect joinder of commands causing validation errors

### PR DESCRIPTION
**Pull request by Trenly**
#21223 and #21225

@denelon 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21228)

### CODE REVIEW

**Review by OfficialEsco — APPROVED** [review]

File: Tools/YamlCreate.ps1 [hunk 1]
```diff
@@ -941,7 +941,7 @@ Function Submit-Manifest {
         git checkout -b "$PackageIdentifier-$PackageVersion" FETCH_HEAD
 
         git add -A
-        git commit -m "$CommitType: `$PackageIdentifier` version $PackageVersion"
+        git commit -m "$CommitType`: `$PackageIdentifier` version $PackageVersion"
```
Comment (OfficialEsco) [inline Tools/YamlCreate.ps1:944]:
```suggestion
        git commit -m "$CommitType`: $PackageIdentifier version $PackageVersion"
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Hold up, this was causing a issue

**Comment by OfficialEsco** [comment]
@denelon need this ASAP

**Comment by Trenly** [comment]
Sorry for all the trouble!

**Review by denelon — APPROVED** [review]

**Comment by denelon** [comment]
@OfficialEsco it looks like you are blocking on a review. Do you want me to merge?

**Review by OfficialEsco — APPROVED** [review]
I am?

**Comment by denelon** [comment]
It just changed.

**Pull request merged**